src/ZendeskApi.Client/IZendeskClient.cs

```diff
@@ -1,8 +1,9 @@
-﻿using ZendeskApi.Client.Resources;
+﻿using ZendeskApi.Client.Http;
+using ZendeskApi.Client.Resources;
 
 namespace ZendeskApi.Client
 {
-    public interface IZendeskClient
+    public interface IZendeskClient : IRestClient
```

Comment:
Yes. ZendeskResource.Client is assigned an IZendeskClient. IZendeskClient is implemented only by ZendeskClient, which implements IRestClient.

Could also fix this by making implementations of ZendeskResource take an IRestClient, but it seems neater to make IZendeskClient extend IRestClient. If you prefer that, happy to change.